decode_test.go

```diff
@@ -298,3 +298,26 @@ func tm(t *testing.T, layout, s string) time.Time {
 	}
 	return tm
 }
+
+func testEncode(t *testing.T) {
+	type S struct {
+		A string "properties:A"
```

Comment:
I think the format of the struct tags is `properties:"<name>"` (with the quotes). I'm surprised this works.

Reply:
checking/changing.....

Reply:
fixed this too